Source/Core/DolphinQt/Settings/AdvancedPane.cpp

```diff
@@ -239,7 +239,7 @@ void AdvancedPane::ConnectLayout()
 
 void AdvancedPane::Update()
 {
-  const bool running = Core::GetState(Core::System::GetInstance()) != Core::State::Uninitialized;
+  const bool running = !Core::IsUninitialized(Core::System::GetInstance());
```

Comment:
[Nit] Everything that uses this does !running so I'd just change it to is_uninitialized instead.